cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -107,14 +109,16 @@
     UPGRADE("Upgrades CDAP to " + ProjectInfo.getVersion() + "\n" +
               "  The upgrade tool upgrades the following: \n" +
               "  1. User and System Datasets (upgrades the coprocessor jars)\n" +
-              "  2. UsageRegistry Dataset Type\n" +
-              "  3. Application Specifications\n" +
-              "  4. Removes all Adapters\n" +
-              "  5. Application Specifications\n" +
+              "  2. Application Specifications\n" +
+              "  3. Removes all Adapters\n" +
+              "  4. Application Specifications\n" +
               "      - Adds artifacts for existing applications\n" +
               "      - Updates application metadata to include newly added artifact\n" +
```

Comment:
Should this be "to include newly added artifacts"? (Sounds like it should be plural)